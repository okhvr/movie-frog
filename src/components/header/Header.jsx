import React, { Component } from 'react';
import { Link } from "react-router-dom";

export default class Header extends Component {

  render() {
    return (
        <nav className="navbar-dark">
            <Link to={'/movies'}>
                <span className="navbar-brand mb-0 h1">moviefrogsroulette</span>
            </Link>
        </nav>
    );
  }
}
