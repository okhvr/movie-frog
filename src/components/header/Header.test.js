import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';
import { BrowserRouter } from "react-router-dom";

describe('Header component', () => {
  it('should match with snapshot', () => {
    const component = renderer.create(
        <BrowserRouter>
            <Header/>
        </BrowserRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});