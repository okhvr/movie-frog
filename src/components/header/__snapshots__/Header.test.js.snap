// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match with snapshot 1`] = `
<nav
  className="navbar-dark"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <span
      className="navbar-brand mb-0 h1"
    >
      moviefrogsroulette
    </span>
  </a>
</nav>
`;
